go.mod

```diff
@@ -1,25 +1,26 @@
 module github.com/infrawatch/sg-core
 
-go 1.14
+go 1.13
```

Comment:
no that's because my dev environment is using 1.13. We should sync this to the build image version and add a CI job that ensures it stays the same in all new incoming PRs

Reply:
It looks like everything I'm checking is referring to golang 1.15 at this point, even when it's on RHEL 8.2 for whatever reason, so I guess we should just sync to golang 1.15.